src/main/java/com/jungdam/emoji/domain/Emoji.java

```diff
@@ -20,8 +22,8 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @Column(name = "emoji_content")
-    private String content;
+    @Embedded
+    private Content content;
 
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "member_Id")
```

Comment:
소문자로 변경해야 합니다! 34번째 줄도 변경 필요해요~
```suggestion
    @JoinColumn(name = "member_id")
```

Reply:
역시 천재 주디!!!